src/v/config/tls_config.h

```diff
@@ -146,23 +167,6 @@ class tls_config {
 };
 
 } // namespace config
-namespace std {
-static inline ostream& operator<<(ostream& o, const config::key_cert& c) {
```

Comment:
excellent. i think was undefined behavior being in `std {`? afaik the only thing allows are template specializations like hash<>

Reply:
There's more: https://github.com/BenPope/redpanda/commit/9a646de7c4df90241c5159538a598b981503fb1b